cmd/check_nomad_long_running/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/jasonhancock/go-nagios"
+)
+
+func main() {
+	fs := flag.CommandLine
+	p := nagios.NewPlugin("nomad-unplaceable", fs)
+
+	p.StringFlag("addr", "http://127.0.0.1:4646", "The address of the Nomad server")
+	p.StringFlag("tls-cert", "", "TLS certificate to use when connecting to Nomad")
+	p.StringFlag("tls-key", "", "TLS key to use when connecting to Nomad")
+	p.StringFlag("tls-ca-cert", "", "TLS CA cert to use to validate the Nomad server certificate")
+	p.BoolFlag("tls-insecure", false, "Whether or not to validate the server certificate")
+	p.StringFlag("days", "2", "If allocation older than X days trigger alert")
```

Comment:
instead of hard-coding it to days, instead take a duration. It appears the github.com/jasonhancock/go-nagios doens't support Durations yet. I'll go get it added